drivers/rtc/rtc_ll_stm32.c

```diff
@@ -123,14 +95,18 @@ static int rtc_stm32_configure(const struct device *dev)
 	if ((hour_format != LL_RTC_HOURFORMAT_24HOUR) ||
 	    (sync_prescaler != cfg->sync_prescaler) ||
 	    (async_prescaler != cfg->async_prescaler)) {
-		err = rtc_stm32_enter_initialization_mode(false);
+		ErrorStatus status = LL_RTC_EnterInitMode(RTC);
+
+		if (status != SUCCESS) {
+			err = -EIO;
+		}
 		if (err == 0) {
```

Comment:
It may be more explicit to replace it by an else no ?

Reply:
I agree, updated.